quinn-udp/benches/throughput.rs

```diff
@@ -72,5 +67,13 @@ pub fn criterion_benchmark(c: &mut Criterion) {
     }
 }
 
+fn new_socket() -> (UdpSocket, UdpSocketState) {
+    let socket = UdpSocket::bind((Ipv6Addr::LOCALHOST, 0))
+        .or_else(|_| UdpSocket::bind((Ipv4Addr::LOCALHOST, 0)))
+        .unwrap();
+
+    let state = UdpSocketState::new((&socket).into()).unwrap();
+    (socket, state)
```

Comment:
Nits: I think this might read better without intermediate bindings, just yielding the tuple directly?

Also, add an empty line below the function please.

Reply:
:+1: done with the latest force push across the 4 commits.